### PR TITLE
Feature/5: Travis CI integration 

### PR DESCRIPTION
**Pull request by zsinnema**
PR for issue #5 
============

Add Continuous Integration (Travis) for master, development and feature branches on PRs

### CODE REVIEW

**Comment by gidili** [comment]
@zsinnema the scripts are looking good, 2 comments: 

1. Can we add the travis badge on the readme as part of this PR too?
2. Should we wait till we resolve that error that makes the build fail before we merge?

**Comment by zsinnema** [comment]
I've added the batches for dev and master
I think we could do the merge, probably will the fix be another feature branch

**Comment by gidili** [comment]
@zsinnema this is good to go 👍

File: pipeline.ipynb [hunk 1]
```diff
@@ -1089,7 +1199,7 @@
     "    bad_units.sort()\n",
     "    if len(bad_units)>0:\n",
     "        hv_res = (hv.NdLayout({\n",
-    "            \"Spatial Footprin\": regrid(hv.Dataset(A_spatial.sel(unit_id=bad_units).rename('A'))\n",
+    "            \"Spatial Footprin\": regrid(hv.Dataset(A_spatial.compute().sel(unit_id=bad_units).rename('A'))\n",
```
Comment (phildong) [inline pipeline.ipynb:1202]: separate commit/PR?

**Comment by zsinnema** [comment]
I will remove pipeline.ipynb from the PR

File: pipeline.ipynb (hunk 1 above)
Comment (zsinnema) [reply pipeline.ipynb]: yes, see below, I've reverted the pipeline.ipynb notebook

**Pull request merged**